tutorials/toolkits/mplot3d.py

```diff
@@ -44,8 +45,6 @@
    :target: ../../gallery/mplot3d/lines3d.html
    :align: center
 
-.. automethod:: Axes3D.plot
```

Comment:
Do we want to replace these with references rather than just deleting?

Reply:
Good idea 👍